src/main/java/janet/Janet.java

```diff
@@ -0,0 +1,126 @@
+package janet;
+
+import java.io.IOException;
+
+import javafx.application.Platform;
+
+/**
+ * Main class of the Janet chat-bot. When the main class is run, it creates an
+ * instance of Janet which allows text-based user interaction.
+ */
+public class Janet {
+    private final Storage storage;
+    private TaskList tasks = new TaskList();
+
+    /**
+     * Class constructor for Janet.
+     */
+    public Janet() {
+        storage = new Storage("janet.txt");
+        try {
+            tasks = storage.readSave();
+        } catch (IOException | ClassNotFoundException e) {
+            System.out.println(e);
+        }
+    }
+
+    /**
+     * Returns a command that represents the user's input in the chat box.
+     *
+     * @param input String entered by the user into the chat box
+     * @return Command that Janet understands
+     */
+    Command getCommandFromInput(String input) {
+        try {
+            return Parser.parseUserInput(input);
+        } catch (JanetException e) {
+            return new Command("echo", e.toString());
+        }
+    }
+
+    /**
+     * Returns the string to be shown in the chat as Janet's reply to user commands.
+     *
+     * @param command Command representing user's input
+     * @return String representing Janet's reply
+     * @throws IOException When unable to write the save
+     */
+    String getResponseFromCommand(Command command) throws IOException {
+        assert(command.getOperation() != null);
+        switch (command.getOperation()) {
+        case "bye":
+            Platform.exit();
+            return "";
+        case "list":
+            return Ui.taskListString(tasks);
+        case "done":
+            return handleTaskDone(command);
+        case "todo":
+            return handleNewToDo(command);
+        case "deadline":
+            return handleNewDeadline(command);
+        case "event":
+            return handleNewEvent(command);
+        case "delete":
+            return handleDelete(command);
+        case "find":
+            return handleFind(command);
+        case "schedule":
+            return handleSchedule(command);
+        case "echo":
+            return command.getDescription();
+        default:
+            return "";
+        }
+    }
```

Comment:
Code very well written with each of the events being handles very well. It becomes clear when that you know what each of the cases does since you have a method to handle each cases.